agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -239,7 +239,7 @@ bool backhaul_manager::socket_disconnected(Socket *sd)
             m_sConfig.slave_iface_socket.erase(iface);
 
             LOG(INFO) << "sending platform_notify: slave socket disconnected " << iface;
-            platform_notify_error(BPL_ERR_BH_SLAVE_SOCKET_DISCONNECTED,
+            platform_notify_error(bpl::BPL_ERR_BH_SLAVE_SOCKET_DISCONNECTED,
```

Comment:
So the error should become `enum class`
```
platform_notify_error(bpl_error::BH_SLAVE_SOCKET_DISCONNECTED;
```
(nitpick)

Reply:
Done.
It's not a nitpicking comment actually: using enum classes over enum types has some advantages that allow for better quality code.